### PR TITLE
fix mask and scale

### PR DESCRIPTION
**Pull request by L1aoXingyu**
这个 PR 主要是修复 Bert 中 mask 的计算问题以及 attention 中的 scale dot product 的问题：

1. 根据党凯在 https://github.com/Oneflow-Inc/libai/pull/36 中的提议，现在将 mask 统一成 mask 的部分是 0，非 mask 部分是 1，这里刚好和 megatron 是反过来的；
2. 王义在对 T5 模型的过程中，在 https://github.com/Oneflow-Inc/libai/pull/54#issue-1101682463 发现 attention 中 qkv 计算完之后，在算完 scaled dot product 之后，进入 ScaleMaskSoftmax 中会被抵消，具体可以看这个 issue
https://github.com/NVIDIA/Megatron-LM/issues/175#issue-1099814485

### CODE REVIEW

File: libai/models/bert_model.py [hunk 1]
```diff
@@ -46,10 +46,7 @@ def forward(self, attention_mask):
         extended_attention_mask = attention_mask_bss.unsqueeze(1)
 
         # Convert attention mask to binary.
-        extended_attention_mask = flow.le(extended_attention_mask, 0.5)
-        # NOTE(Lxy): '<' is not work!
-        # extended_attention_mask = (extended_attention_mask < 0.5)
-
+        extended_attention_mask = extended_attention_mask > 0.5
```
Comment (marigoold) [inline libai/models/bert_model.py:49]: 这个之前NOTE "< is not work" 现在可以了吗？

Comment (L1aoXingyu) [reply libai/models/bert_model.py:49]: 这个 > 运算是正常的，你可以试试看

**Review by marigoold — APPROVED** [review]
